File: PHP/uberX.php
<?php 
require_once ('car.php');
    
class UberX extends Car{
          public $brand;
          public $model;


          //constructor
          public function __construct($license, $driver, $brand, $model){
               parent::__construct($license, $driver); //equivale la palabra super en ja, $brand, $modelva

               $this->license = $license;
               $this->dirver = $driver;
               $this->brand = $brand;
               $this->model = $model;
          }
     }


?>

File: JAVA/UberX.java
class UberX extends Car {
     String brand;
     String model;

     // constructor que debe councidir con la clase padre
     public UberX(String license, Account driver, String brand, String model) {
          // para referirme al metodo controstructor de la clase padre(Car)
          super(license, driver);

          this.brand = brand;

          this.model = model;
     }

     @Override
     void printDataCar() {
          super.printDataCar();
          System.out.println("Modelo: " + model + "Brand: " + brand);

     }

}